### PR TITLE
Hotfix: Bridge Minimum amount and shuttle fee

### PR DESCRIPTION
**Pull request by mystbrent**
## Description of Changes

All of this code will be superseded in a WIP branch I have in my local. Just added hotfix to complement the current SLS bridge fee structure for the interest of time

1. Added button validation for minimum token amount
2. Updated shuttle fee of 20 for every token

### Developer Checklist:

* [ ] I have followed the guidelines in our Contributing document
* [ ] This PR has a corresponding JIRA ticket
* [ ] My branch conforms with our naming convention i.e. `feature/HDEV-XXX-description`
* [ ] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [ ] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

File: src/utils/bridge.ts [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
-  return (amount * Number(process.env.REACT_APP_SHUTTLE_FEE_PERCENTAGE ?? 0)).toFixed(18).replace(/\.?0+$/, '')
+// export function calculateShuttleFee(amount: number) {
+//   return (amount * Number(process.env.REACT_APP_SHUTTLE_FEE_PERCENTAGE ?? 0)).toFixed(18).replace(/\.?0+$/, '')
+// }
+
+export function calculateShuttleFee(tokenSymbol: string) {
+  if (tokenSymbol.startsWith('w')) {
+    return 20
+  }
+  return 1
```
Comment (0xAplki) [inline src/utils/bridge.ts:9]: so shuttle fee is always one?

**Review by 0xAplki — COMMENTED** [review]
just one q just to understand things but im actually good with this, we'll just wait for the order of merging

File: src/utils/bridge.ts (hunk 1 above)
Comment (mystbrent) [reply src/utils/bridge.ts:9]: This is temporary - The goal is on MATIC -> ETH tx fee would be 20 of whatever tokens. On ETH -> MATIC we would charge only 1 of token.

So if its from MATIC it starts with 'w' hence the hacked code this is just temporary

**Review by tracyarciaga — APPROVED** [review]
LGTM

File: src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx [hunk 2]
```diff
@@ -156,11 +156,12 @@ const ConfirmTransactionModal = ({
           <b>
             {amount} {tokenSymbol}
           </b>{' '}
-          for{' '}
+          {console.log('wrappedTokenSymbol:', wrappedTokenSymbol)}
```
Comment (schystz) [inline src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx:159]: Shall we remove this @mystbrent ?

File: src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx [hunk 3]
```diff
@@ -156,11 +156,12 @@ const ConfirmTransactionModal = ({
...
+          {/* for{' '}
```
Comment (schystz) [inline src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx]: Can we remove commented code?

File: src/utils/bridge.ts [hunk 4]
```diff
@@ -1,3 +1,10 @@
-export function calculateShuttleFee(amount: number) {
-  return (amount * Number(process.env.REACT_APP_SHUTTLE_FEE_PERCENTAGE ?? 0)).toFixed(18).replace(/\.?0+$/, '')
+// export function calculateShuttleFee(amount: number) {
```
Comment (schystz) [inline src/utils/bridge.ts:1]: Can we remove the commented code?

File: src/utils/bridge.ts (hunk 1 above)
Comment (schystz) [reply src/utils/bridge.ts:9]: Would it be better to check `chainId` instead if MATIC or MAINNET? Just thinking what would be a more bullet proof condition.

**Review by schystz — COMMENTED** [review]
Just some minor comments @mystbrent 

Also the gas estimate issue I mentioned on slack, should it be an easy fix otherwise let's create a separate ticket

**Comment by mystbrent** [comment]
https://github.com/HaloDAO/halodao-interface/pull/80#discussion_r708122734

@schystz Yes I agree and that was actually I am doing in the WIP branch. This is hotfix is really temporary since I am still working on the SLS bridge for the updated fee structure. But the goal is actually to charge based on the estimated gas fees paid by relayer (SLS bridge)

**Review by schystz — APPROVED** [review]

**Pull request merged**